src/services/data_model_engine.js

```diff
@@ -274,6 +274,14 @@ export default class DataModelEngine {
     await this.bundleRepository.setBundleRepository(bundleId, BundleStatuses.sheltered, {holdUntil: new Date(bundleExpirationDate)});
   }
 
+  /**
+   * @returns {Promise<void>}: removed bundles count
```

Comment:
I think it's `Promise<int>` if it returns count.

Reply:
👍